### PR TITLE
Gulp build hanging

### PR DESCRIPTION
**Pull request by devel-pa**
#32 fix + browserify-swap issue on windows.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -35,7 +41,10 @@ gulp.task('styles', function() {
 });
 
 var bundler = _.memoize(function() {
```
Comment (megawac) [inline gulpfile.js]: Will this cause issues

**Comment by devel-pa** [comment]
The issue is caused by the fact that watchify is used for build, too. 
In the way the gulpfile is done memoize is doing its job.
Refactoring the file with each task in a separate file and a config file for folders and files can be done later.

**Comment by devel-pa** [comment]
Probably that area should be rewritten and create a bit more complex config based on gulp-if with a task for production.

**Comment by rsumilang** [comment]
I didn't notice this pull request when making my own hehe. I did my own version of the fix (#45) that doesn't current build tasks.

**Comment by thejameskyle** [comment]
Closing in favor of #45

**Pull request closed without merging**